README.md

```diff
@@ -119,6 +119,12 @@ However, you can also deploy Jupyter Hub separately from the servers it
 deploys (e.g. Jupyter Lab or Cylc UI Server) and launch it via the `jupyterhub`
 command.
 
+Enable the Cylc UI Server in `jupyterhub` or `jupyter lab` by running
+```bash
+jupyter server extension enable --py cylc.uiserver
+```
+The Cylc UI will be available by replacing `/lab` in the URL with `/cylc`.
+
```

Comment:
```suggestion
```

I think this command will fail (or create a file in a user-specific location) in many situations because it requires the user to have write access to the deployment which they might not have. For transparency we aught to document the configuration and leave them to decide where best to configure it.

Note, this applies to Jupyter Lab *not* Jupyter Hub.